keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceCheckScheduler.kt

```diff
@@ -34,6 +36,7 @@ class ResourceCheckScheduler(
   @Scheduled(fixedDelayString = "\${keel.resource-check.frequency:PT1S}")
   fun checkResources() {
     if (enabled) {
+      publisher.publishEvent(ScheduledCheckStarting)
```

Comment:
Why the change to publishing an event here?

Reply:
Needed something for the test to listen to to confirm the scheduler does not hang.

I figure tacking a health check to this probably makes sense too.